docs/sources/static/configuration/integrations/snmp-config.md

```diff
@@ -15,7 +15,7 @@ which is an embedded version of
 [`snmp_exporter`](https://github.com/prometheus/snmp_exporter). This allows collection of SNMP metrics from the network devices with ease.
 
 {{< admonition type="note" >}}
-`snmp config` uses the latest configuration introduced in version 0.23 of the Prometheus `snmp_exporter`.
+`snmp config` uses the latest configuration introduced in version 0.26 of the Prometheus `snmp_exporter`.
```

Comment:
Unfortunately, this would be a breaking change and I cannot add it right now. I added an [issue](https://github.com/grafana/alloy/issues/916) for us to fix it for Alloy v2.

The good news is that I think you can set `config_file` using a glob pattern like `snmp_files*`. I'm not going to document it right now though, because I'm not 100% sure how to test it.  I'll raise an issue for this and other such items to be documented and tested.